securedrop_client/gui/widgets.py

```diff
@@ -1157,7 +1159,7 @@ class ConversationWidget(QWidget):
     def __init__(self,
                  message_id: str,
                  message: str,
-                 update_signal,
+                 update_signal: Any,
```

Comment:
`Any` -> `Function[...]` (you'll have to double check the args here).

Reply:
I tried grepping for where ConversationWidget was instantiated, and I only found the tests where the signal was made by Mocker. I looked up PyQt5 stackoverflow, documentation, and source code for what type signals are, but I realize I have no clue what I'm doing.
For now I'll do exactly as you say since I think I don't know what I'm double checking.

Reply:
Nevermind. I'm catching on.

Reply:
Nevermind the nevermind. I'm now back to the issue that I don't know the arguments or return because the only place I see these called are by each other or by tests using mocker.

Reply:
You can look in the PyQt5 docs, or just load stuff into the interpreter and use `type()` to figure it out.